.github/workflows/ci-pr.yml

```diff
@@ -0,0 +1,119 @@
+name: CI-PR
+on:
+  pull_request:
+    branches: [master]
+concurrency:
+  group: ci-${{ github.head_ref }}
+  cancel-in-progress: true
+jobs:
+  ci-pr:
+    name: CI-PR
+    needs: [test, msrv, lint]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Done
+        run: exit 0
+  test:
+    name: Tests
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "windows-latest"]
+        rust: ["stable"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ matrix.rust }}
+          override: true
+      - uses: Swatinem/rust-cache@v1
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Default features
+        run: cargo test --all-targets
+      - name: All features + Debug
+        run: cargo test --all-targets  --features "wrap_help yaml regex debug"
+      - name: No features
+        run: cargo test --all-targets --no-default-features --features "std cargo"
+  msrv:
```

Comment:
If we want to test stable, we can make this step stable. But I honestly don't feel like we should need another step. Let's just `stable` to rust matrix in the above step.

Reply:
I'm waiting on further discussion on this based on resolving the other discussion of focusing on stable vs MSRV